services/web/server/src/simcore_service_webserver/projects/db.py

```diff
@@ -436,90 +436,96 @@ async def replace_project(
         product_name: str,
         project_uuid: str,
     ) -> dict[str, Any]:
-        """replaces a project from a user
+        """DEPRECATED!!
```

Comment:
Can we have a reason for the deprecation here?

Reply:
no. the next PR will bring this in.